.circleci/config.yml

```diff
@@ -129,7 +129,7 @@ jobs:
           name: integration tests
           command: npm run int-test
       - store_test_results:
-          path: test-results
+          path: test_results
```

Comment:
left over from a previous change to bring in line with typescript template project that uses `test_results` instead